src/utils/daos_autotest.c

```diff
@@ -27,8 +27,26 @@
 /** Input arguments passed to daos utility */
 struct cmd_args_s *autotest_ap;
 
+/* The start and end time in ns for individual test*/
+uint64_t	t_start, t_end;
+char		iops[32];
 /** How many concurrent I/O in flight */
 #define MAX_INFLIGHT 16
+#define OUTPUT_IOPS(x) ({\
+		/* negative value is unexpected*/\
+		if (x < 0.0) {\
+			snprintf(iops, sizeof(iops)-1, "%7.2leG IO/sec", (x)*0.000001);\
+		} else if (x < 1000.0) {\
+			snprintf(iops, sizeof(iops)-1, "%7.2lfK IO/sec", (x));\
+		} else if (x < 1000000.0) {\
+			snprintf(iops, sizeof(iops)-1, "%7.2lfM IO/sec", (x)*0.001);\
+		} else if (x < 1000000000.0) {\
```

Comment:
Why do you not merge this case with the previous one (just one else) ?

Reply:
The print format is adaptively selected to print in unit K/M/G/T now. Smaller range makes the number easier to read.